packages/logger/index.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: @loopback/logger
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+// NOTE(bajtos) This file is used by VSCode/TypeScriptServer at dev time only
+export * from './src';
```

Comment:
I think you will need to add `packages/logger/index.js` and `packages/logger/index.d.ts` with the same content as we have in other projects.